### DISCUSSION TITLE
Welcome to Antura Discussions!

### DISCUSSION THREAD

**Discussion by StefanoCecere** [vgwb/Antura#98] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KevinSaisai** [comment]
I found and downloaded the iOS version of Antura, but there is no Chinese (Mandarin) option in the native language settings, which makes it quite difficult for me to use.

**Reply by KevinSaisai** [reply]
This is a really great game! Even though there’s no Chinese (Mandarin) option for the native language, I still played it for over ten minutes and had a fantastic experience. I originally wanted to play and learn with my child, but it’s too difficult for her.

**Reply by StefanoCecere** [reply]
hi @KevinSaisai chinese is a native language that we'll support in the near future, for sure!
did you play the literacy games or the new discovery ones (france and poland)?
